sematic/db/models/run.py

```diff
@@ -194,19 +184,6 @@ def external_exception_metadata(
             exception_metadata
         )
 
-    @property
-    def external_jobs(self) -> Tuple[ExternalJob, ...]:
-        """Representations of the external compute jobs used for the run."""
-        encodables = self.external_jobs_json
-        encodables = encodables if encodables is not None else []
-        return tuple(value_from_json_encodable(job, ExternalJob) for job in encodables)
-
-    @external_jobs.setter
-    def external_jobs(self, jobs: Sequence[ExternalJob]) -> None:
-        self.external_jobs_json = [
-            value_to_json_encodable(job, ExternalJob) for job in jobs
-        ]
-
```

Comment:
Can you use `sqlalchemy.orm.relationship` to declare the relationship.
https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html